js/login.js

```diff
@@ -0,0 +1,41 @@
+/* variables */
+
+let logIn = document.getElementById("login-btn");
+let loginForm = document.querySelector(".login-form");
+
+logIn.addEventListener("click", (e) => {
+    e.preventDefault();
+
+
+    let emailId = document.getElementById('email').value;
+    let userPwd = document.getElementById('password').value;
+    const userData = JSON.parse(localStorage.getItem('userData')) || [];
+  
+
+
+let keyProp = Object.keys(userData);
+
+let propertyValue = Object.values(userData);
+
+let email, passwd;
+let found = "n";
+
+for(let i = 0; i < keyProp.length; i++){ 
+        email = propertyValue[i].userEmail;
+        passwd = propertyValue[i].password;
+        console.log(email);
+        console.log(passwd);
+        
+        if( passwd === userPwd && email === emailId) {
+            window.open("./index.html", target="_self");
+            found = "y";
+            break;
+        } 
+}
+if(found === "n"){
```

Comment:
```suggestion
        if(checkCredentials(emailId, userPwd, userData)) {
            window.open("./account.html");
        } else {
          alert("Invalid Login Credentials!");
}
```
The checkCredentials function is called inside the if statement. If it returns true, the window.open() method will open ./account.hmtl. Since index.html holds our main page, we don't want to blocked it off behind a login page. Instead, account creation can be an optional feature, instead of a requirement. For the window.open() method, "_self" is the default behavior, so we can remove the second argument.